packages/core/src/components/private/ButtonOrLink.tsx

```diff
@@ -78,7 +78,7 @@ export default class ButtonOrLink extends React.Component<Props> {
 
       if (openInNewWindow) {
         props.target = '_blank';
-        props.rel = 'noopener noreferrer';
+        props.rel = rel === undefined ? 'noopener noreferrer' : rel;
```

Comment:
Where does `rel` come from?

Reply:
Whoops, sorry!  Had some git trouble, and it got lost.

Reply:
Be sure to set rel again when `openInNewWindow` is not being used.